addons/pos_self_order/models/pos_config.py

```diff
@@ -69,12 +69,10 @@ def _self_order_kiosk_default_languages(self):
             [("code", "=", self.env.lang)], limit=1
         ),
     )
-    self_order_kiosk_image_home = fields.Image(
-        string="Self Order Kiosk Image Home",
+    self_order_kiosk_image_home = fields.Many2many(
```

Comment:
Change the name of this to understand that it is many_to_many

> 
```suggestion
    self_order_kiosk_image_home_ids = fields.Many2many(
```